### PR TITLE
Better tests

### PR DESCRIPTION
**Pull request by jonasfj**
Just some small improvements... and hopefully some improved coverage... once I get automation for QEMU tests (not now as I can't do it in a VM which is all travis offers)...

### CODE REVIEW

File: commands/qemu-build/buildimage.go [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+	"github.com/taskcluster/taskcluster-worker/engines/qemu/network"
+	"github.com/taskcluster/taskcluster-worker/engines/qemu/vm"
+)
+
+func buildImage(
+	log *logrus.Entry,
+	inputFile, outputFile string,
+	fromImage, novnc bool,
+	boot,
+	cdrom string,
```
Comment (petemoore) [inline commands/qemu-build/buildimage.go]: Join lines 22 and 23? Alternatively maybe one parameter per line?

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,4 @@
 /taskcluster-worker
 /.vagrant
 coverage.out
+/.bash_history
```
Comment (petemoore) [inline .gitignore]: How does .bash_history get added to git repo?

File: commands/qemu-build/buildimage.go [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+	log.Info("Creating user-space network")
+	net, err := network.NewUserNetwork(tempFolder)
+	if err != nil {
+		log.Error("Failed to create user-space network, error: ", err)
+		return err
+	}
+
+	// Create virtual machine
+	log.Info("Creating virtual machine")
+	vm := vm.NewVirtualMachine(img, net, socketFolder, boot, cdrom, log.WithField("component", "vm"))
```
Comment (petemoore) [inline commands/qemu-build/buildimage.go]: It is marginally confusing (although I guess allowed) to have the variable match the package name.

**Comment by petemoore** [comment]
lgtm r+

File: .gitignore (hunk 2 above)
Comment (jonasfj) [inline .gitignore]: It's from when we run `docker-test.sh` I touch and mount it in locally.. So it remembers commands inside the docker environment... Just a neat trick..

**Pull request merged**